app/views/directives/osc-secrets.html

```diff
@@ -0,0 +1,34 @@
+<ng-form name="secretsForm" class="osc-secrets-form">
+  <div class="form-group">
+
+    <label class="picker-label">{{displayType | startCase}} Secret</label>
+    <ui-select ng-model="pickedSecret.name">
+      <ui-select-match placeholder="Secret name">{{$select.selected}}</ui-select-match>
+      <ui-select-choices repeat="secret in (secretsByType[type] | filter : $select.search)">
+        <div ng-bind-html="secret | highlight : $select.search"></div>
+      </ui-select-choices>
+    </ui-select>
+    <div ng-switch="displayType">
+      <div class="help-block" ng-switch-when="source">
+        Name of a secret that contains credentials used for pulling your source.
```

Comment:
I'd remove "Name of a ..." from these and just say "Secret with credentials for pulling your source code."